sdk/monitor/azquery/README.md

```diff
@@ -102,6 +102,9 @@ The timespan can be the following string formats:
 <duration> such as "P1Y2M10DT2H30M" // 1 year, 2 months, 10 days, 2 hours, 20 minutes
 ```
 
+### Logs Workspace IDs
```

Comment:
Is this section supposed to be empty?

Reply:
It's not. Thanks for catching! Will fix.

Reply:
fixed in commit [d5e54b6](https://github.com/Azure/azure-sdk-for-go/pull/19208/commits/d5e54b6debc04da260e9cf6b8205c4565b57c56a)